### ISSUE TITLE
Google search / All websites

### ISSUE THREAD

**Issue by srikanthlogic** [yuvipanda/yenWikipedia#9]
Done in FF, provide option in Chrome port. But let Google search alone be default in both cases 
@yuvipanda : thoughts?

**Comment by yuvipanda** [comment]
Not sure if that's a good idea. Needs a good UX. We can't just add it to the right as we do now.

**Comment by srikanthlogic** [comment]
agreed, probably on other sites, we can replace the title by word "language name" in that specified lang for other sites, which may not affect the UX on other sites much

**Comment by tecoholic** [comment]
Thinking of just adding a down arrow near to the original link titled "Other Languages", which would expand to show the links.
Suggestions: @srikanthlogic

**Comment by srikanthlogic** [comment]
@tecoholic we must use a extn standard icon. Yeah, so replace down arrow with icon and give a build, will test.
